<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>UserDataRequest</class>
  <widget class="QDialog" name="UserDataRequest" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>216</width>
        <height>103</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Authentication credentials required</string>
    </property>
    <property name="sizeGripEnabled" >
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <widget class="QLabel" name="queryInfo" >
          <property name="text" >
            <string/>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="queryField" >
              <property name="text" >
                <string/>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="queryEdit" >
              <property name="enabled" >
                <bool>true</bool>
              </property>
              <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer4" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction></pixmapfunction>
</ui>
